_rules/audio-text-alternative-e7aa44.md

```diff
@@ -32,9 +32,9 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is:
+This rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that:
```

Comment:
Can use the ["listed condition"](https://act-rules.github.io/pages/design/rule-design/#listed-conditions) syntax (since we update the rule):
```suggestion
This rule applies to any [non-streaming](#non-streaming-media-element) `audio` element for which at least one of the following is true:
```